packages/bot-web-ui/src/app/app-content.jsx

```diff
@@ -59,9 +63,9 @@ const AppContent = observer(() => {
     }, [client.is_options_blocked, client.account_settings.country_code]);
 
     React.useEffect(() => {
-        GTM.init(DBotStores);
+        GTM.init(combinedStore);
```

Comment:
@shafin-deriv  why are we changing from `DBotStores` to `combinedStore`

Reply:
DBotStore currently contains the property "core" and other Dbot specific stores. But once we refactor the RootStore and remove the core properties (ex: client, common, notification, etc) from RootStore then this will break the current implementation of MobxContentProvider. Eventually once we remove the connect method then we can remove this MobxContentProvider as well and use the useDBotStore() hook for getting the DBot specific stores and useStore() hook for getting the root specific stores ('@deriv/stores')